index.js

```diff
@@ -63,6 +63,14 @@ class ServerlessCustomDomain {
     });
   }
 
+  setGivenDomainName() {
+    /**
+     * Since serverless version 1.13.0, cli variables in the serverless file
+     * are not converted until before deploy initialize.
+      */
+    this.givenDomainName = this.serverless.service.custom.customDomain.domainName;
```

Comment:
Then I would have to set it in every function that it is used in. This way I don't have to repeat setting the variables. If you feel repeating it is better, I'll change it.